script/exifautotran

```diff
@@ -26,11 +26,11 @@ do
   esac
 
   if test -n "$transform"; then
-    convert $transform $file tempfile.jpg
+    convert $transform $file temp-$file.jpg
```

Comment:
Just one consideration -- are you certain $file doesn't contain any directories?  I'll check....  Shoot, you can't tell from this script, depends how it's called.  How about using the pid instead?  temp-$$.jpg  Does that work in bourne shell?  Yes, it does.  That way it won't matter how it's called.

I'll check the process_image script to see if there are any other temp files with similar issues.

Reply:
PS. exifautotran is definitely be called with a relative path not just a file name.  image_root/orig/id.jpg.  Looks like this is the only tempfile used in the entire process_image script.  Although, the initial file from nginx may be a temp file, but I'm sure rails handles that intelligently.  This exifautotran script was quick and dirty, not surprised it was missing something so critical.